examples/verif-manual/vm-291.py

```diff
@@ -0,0 +1,574 @@
+"""
+VM291 Force on the Boundary of a Semi-Infinite Body (Boussinesq Problem)
+========================================================================
+
+Description:
+A point force is applied at the origin of a half-space 2-D axisymmetric
+solid modeled with far-field domain. Determine the displacement in the
+Y-direction on nodes along the radial direction (at location Y = 0)
+and vertical direction (at location X = 0).
+
+"""
+import os
+
+# Importing the `launch_mapdl` function from the `ansys.mapdl.core` module
+from ansys.mapdl.core import launch_mapdl
+
+# Launch MAPDL with specified options
+mapdl = launch_mapdl(loglevel="WARNING", print_com=True, remove_temp_dir_on_exit=True)
+
+# Clear the current database
+mapdl.clear()
+
+# Run the FINISH command to exists normally from a processor
+mapdl.run("FINISH")
+
+# Set the ANSYS version
+mapdl.com("ANSYS MEDIA REL. 2022R2 (05/13/2022) REF. VERIF. MANUAL: REL. 2022R2")
+
+# Run the /VERIFY command for VM291
+mapdl.run("/VERIFY,VM291")
+
+# Set the title of the analysis
+mapdl.title("VM291FORCE ON BOUNDARY OF A SEMI-INFINITE BODY (BOUSSINESQ PROBLEM)")
+
+# Comment line: Providing additional information about the analysis
+mapdl.com("")
+mapdl.com("REFERENCE: 'TIMOSHENKO,S.P.,AND J.N.GOODIER,THEORY OF ELASTICITY")
+mapdl.com("MCGRAW-HILL,NEW YORK, PP 398-402, 1970")
+mapdl.com("******************************************")
+mapdl.com("USING PLANE182 AND INFIN257 ELEMENTS")
+mapdl.com("*******************************************")
+
+# Entering the PREP7 environment in MAPDL
+mapdl.prep7()
+
+# Constant value of PI
+mapdl.run("PI=ACOS(-1)")
+
+# 2D 4-NODE STRUCTURAL SOLID
+mapdl.et(1, "PLANE182")
+# Set keyopt(3)=1, AXISYMMETRIC
+mapdl.keyopt(1, 3, 1)
+
+# DEFINE MATERIAL MODEL
+# YOUNG'S MODULUS
+mapdl.run("Exx=1.0                 ")
+mapdl.mp("EX", 1, "Exx")
+# POISSON'S RATIO
+mapdl.run("NUxy=0.1                ")
+mapdl.mp("PRXY", 1, "NUxy")
```

Comment:
It would be nicer to directly use Python instead of MAPDL to store the variables.

```suggestion
# Constant value of PI
pi = math.pi # need to add "import math" at the beginning of the file

# 2D 4-NODE STRUCTURAL SOLID
mapdl.et(1, "PLANE182")
# Set keyopt(3)=1, AXISYMMETRIC
mapdl.keyopt(1, 3, 1)

# DEFINE MATERIAL MODEL
# YOUNG'S MODULUS
exx=1.0
mapdl.mp("EX", 1, exx)
# POISSON'S RATIO
nuxy=0.1
mapdl.mp("PRXY", 1, nuxy)
```